eth/eventdatahandler/task_executor_test.go

```diff
@@ -24,8 +25,8 @@ const rawValidatorAdded = `{
 		"transactionHash": "0x921a3f836fb873a40aa4f83097e52b69225334c49674dc262b2bb90d27e3a801"
 	  }`
 
-func TestExecuteTask(t *testing.T) {
-	logger := zaptest.NewLogger(t)
+func TestExecuteStartValidatorTask(t *testing.T) {
+	logger, logs := setupLogsCapture()
```

Comment:
what do you think of `observedLogs`?
```suggestion
	logger, observedLogs := setupLogsCapture()
```